source/core/TypeAndDataManager.cpp

```diff
@@ -2876,6 +2894,22 @@ VIREO_FUNCTION_SIGNATURE4(TypeGetSubElementFromPath, StaticType, void, StringRef
   //  tm->PointerToSymbolPath(_Param(1), _Param(2), _Param(3));
     return _NextInstruction();
 }
+//------------------------------------------------------------
+//! Parse through a path from a starting point to target subelement, return subelement value if it matches expected output type.
+VIREO_FUNCTION_SIGNATURE5(GetSubElementFromPath, StaticType, void, StringRef, StaticType, void)
```

Comment:
No, why would we here?  There's a starting object (usually the VI), a string path, and a destination object.  Nothing's optional.